### PR TITLE
fixes #68228 Clarify width and height properties in Image component docs

### LINKED ISSUE #68228: Docs: Clarity on the width and height properties of the Image component
### What is the update you wish to see?

Might be a mistake on my part, but I feel like the docs are in need of a bit of clarity on the width and height attributes of the Image component?

[Here](https://nextjs.org/docs/pages/api-reference/components/image#width) they're defined as representing the rendered sizes:

> The width property represents the rendered width in pixels, so it will affect how large the image appears.

But [here](https://nextjs.org/docs/pages/building-your-application/optimizing/images#remote-images) it states that they do not:

> The width and height attributes are used to infer the correct aspect ratio of image and avoid layout shift from the image loading in. The width and height do not determine the rendered size of the image file.

Feel like it should be the latter, the intrinsic values, so that it's inline with the `` tag?

### Is there any context that might help us understand?

As above

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/pages/api-reference/components/image#width

### PR DESCRIPTION
**Pull request by Joe-zzj**
### Clarify width and height properties in Image component docs

This PR updates the documentation for the `width` and `height` properties of the Image component to clarify that these attributes represent the intrinsic dimensions of the image and do not determine the rendered size, which is controlled by CSS. This clarification aligns the documentation with the behavior of the HTML `` tag.

Fixes issue #68228.
Closes https://github.com/vercel/next.js/issues/68228

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: e85bbf8ecff7daa9bb48653bea2cef4b06aca2d8

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: e85bbf8ecff7daa9bb48653bea2cef4b06aca2d8

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: e85bbf8ecff7daa9bb48653bea2cef4b06aca2d8

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/02-api-reference/01-components/image.mdx [hunk 1]
```diff
@@ -93,13 +93,13 @@ When using an external URL, you must add it to [remotePatterns](#remotepatterns)
 
 ### `width`
 
-The `width` property represents the _intrisic_ image width in pixels.
+The `width` property represents the _intrisic_ image width in pixels.This property is used to infer the correct aspect ratio of the image and avoid layout shift during loading. It does not determine the rendered size of the image, which is controlled by CSS, similar to the `width` attribute in the HTML `<img>` tag.
```
Comment (Willem-Jaap) [inline docs/02-app/02-api-reference/01-components/image.mdx:96]:
```suggestion
The `width` property represents the _intrisic_ image width in pixels. This property is used to infer the correct aspect ratio of the image and avoid layout shift during loading. It does not determine the rendered size of the image, which is controlled by CSS, similar to the `width` attribute in the HTML `<img>` tag.
```

File: docs/02-app/02-api-reference/01-components/image.mdx [hunk 2]
```diff
@@ -93,13 +93,13 @@ When using an external URL, you must add it to [remotePatterns](#remotepatterns)
...
 
 Required, except for [statically imported images](/docs/app/building-your-application/optimizing/images#local-images) or images with the [`fill` property](#fill).
 
 ### `height`
 
-The `height` property represents the _intrisic_ image height in pixels.
+The `height` property represents the _intrisic_ image height in pixels.This property is used to infer the correct aspect ratio of the image and avoid layout shift during loading. It does not determine the rendered size of the image, which is controlled by CSS, similar to the `height` attribute in the HTML `<img>` tag.
```
Comment (Willem-Jaap) [inline docs/02-app/02-api-reference/01-components/image.mdx:102]:
```suggestion
The `height` property represents the _intrisic_ image height in pixels. This property is used to infer the correct aspect ratio of the image and avoid layout shift during loading. It does not determine the rendered size of the image, which is controlled by CSS, similar to the `height` attribute in the HTML `<img>` tag.
```

**Review by Willem-Jaap — CHANGES_REQUESTED** [review]

**Review by ijjk — APPROVED** [review]

**Pull request merged**